airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/ConnectionName.tsx

```diff
@@ -130,9 +42,16 @@ const ConnectionName: React.FC<Props> = ({ connection }) => {
   };
 
   const updateConnectionAsync = async () => {
-    // Update only when the name is changed
-    if (connection.name !== connectionName) {
+    if (connectionName === undefined || connection.name === connectionName.trim() || !connectionName.trim()) {
+      setConnectionName(connectionNameBackup);
+      setEditingState(false);
+      return;
+    }
+
+    const connectionNameTrimmed = connectionName.trim();
```

Comment:
You could use this var as the check:
```suggestion
    const connectionNameTrimmed = connectionName?.trim();
    if (!connectionNameTrimmed || connection.name === connectionNameTrimmed) {
      setConnectionName(connectionNameBackup);
      setEditingState(false);
      return;
    }
```

Reply:
Yeah, you are right! I will update this part 👍